pkg_defs/ska_sun/meta.yaml

```diff
@@ -20,16 +20,20 @@ requirements:
     - setuptools_scm
     - setuptools_scm_git_archive
     - ska_helpers
+    - testr
   run:
     - python
     - ska.quatutil
     - chandra.time
     - quaternion
     - numpy
+    - ska_helpers
+    - testr
```

Comment:
`ska_helpers` is not a runtime dependence, right? And `testr` is an optional runtime dependence, so I don't know where we stand on optional dependencies in this manifest.

Or do the `run` packages here relate to building in any way?

Reply:
I thought ska_helpers was a runtime dependence if you want to guarantee you can get the version (with ska_helpers.get_version) from an environment where this is installed.

Reply:
And I didn't consider "ability to run tests" to be optional, so I thought this was a defect in the original.

Reply:
OK, you're right. Sounds fine.